README.md

```diff
@@ -1,2 +1,14 @@
 # SUSE Enceladus
 
+## Introduction
+
+The SUSE-Enceladus organization in GitHub was created to serve the needs of the SUSE Public Cloud DevOps team. We started out with a "collector" project named Enceladus within the SUSE organization. Over time the collector project grew unwieldy and we had various unrelated source bases intermingled in the Enceladus project. In addition some of our other projects, large an small were maintained in separate projects in separate orgs. SUSE-Enceladus was created to collect our efforts for original source code in one organization, help us be more organized, and provide a better overview of the projects we develop and maintain. The initial projects in this organization started as migrations from Enceladus or other projects we maintained in GitHub.
+
+## Goals
+
+We strive to provide a reasonably uniform experience for all projects within the SUSE-Enceladus organization. In an effort to achieve this uniform experience we strive to document our processes and procedures. Each project will have a LICENSE, a README.md, and a CONTRIBUTING.md file that provides information about he project. However, projects do have different needs and therefore some differences are to be expected.
+
+Projects in the SUSE-Enceladus organization represent original work of the SUSE Public Cloud DevOps team. We hope our work is useful to you and we welcome any contribution. The project is in alignment with the SUSE open source policy [1] and as such we do not have a contributor agreement.
```

Comment:
"The project is in alignment..." On the first read I thought "which project", so this covers all projects in the SUSE-Enceladus organisation correct ?